src/app/public/modules/datepicker/datepicker.component.ts

```diff
@@ -31,29 +37,39 @@ import {
   styleUrls: ['./datepicker.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class SkyDatepickerComponent implements OnDestroy {
+export class SkyDatepickerComponent implements OnDestroy, OnInit {
+
   /**
    * @internal
    * Indicates if the calendar button element or any of its children have focus.
    */
   public get buttonIsFocused(): boolean {
```

Comment:
Do you know where this public method is being used? If it's being used by other libraries (or our consumers) we need to slap a `@deprecated` message on since the components won't be accessible in SKY UX 4. I had to do the same thing with the popovers library.

In the case of popovers, Erika had created public methods on the component so she could get the component's state within ag-grid. We'll need to figure out a different solution for SKY UX 4.

Reply:
Yep, that's the exact same scenario. These are for ag-grid. I'll add `@deprecated` to them.